### PR TITLE
Use pkgconf instead of pkg-config in Solus

### PR DESCRIPTION
**Pull request by ur5us**
Solus migrated from pkg-config to the more modern pkgconf, s.
https://github.com/getsolus/packages/commit/0949eab4adfffebb988fd0c8448c52fcddd82969#diff-6da1d22dcf408888f8aa5336844f716dd18fac4d39c0be32f5011b9828a6af53R35-R37

### CODE REVIEW

**Comment by torrocus** [comment]
Could you please add a short description of your changes to the [CHANGELOG](https://github.com/rvm/rvm/pull/CHANGELOG.md)?

**Comment by ur5us** [comment]
@torrocus Done.

File: CHANGELOG.md [hunk 1]
```diff
@@ -34,6 +34,7 @@
 * Fix gentoo dependencies moved from sys-devel to dev-build [\#5432](https://github.com/rvm/rvm/pull/5432)
 * Fix truffleruby-head installation on macos-aarch64 [\#5446](https://github.com/rvm/rvm/pull/5446)
 * Fix bug with Bootsnap on Ruby 3.3.1 [\#5457](https://github.com/rvm/rvm/pull/5457)
+* Fix Solus requirements check for pkgconf
```
Comment (torrocus) [inline CHANGELOG.md:37]:
```suggestion
* Fix Solus requirements check for pkgconf [\#5471](https://github.com/rvm/rvm/pull/5471)
```

**Review by pkuczynski — APPROVED** [review]
Is this a recent change? Shall we maybe make it backwards compatible to keep support for older versions of Solus?

**Comment by ur5us** [comment]
@pkuczynski Yes, it's a recent change, s. my link. Solus is a rolling release distribution. Not sure about the value of backwards compatability given its popularity.

**Comment by pkuczynski** [comment]
If it's rolling,than it doesn't really make sense, true. If someone reports a bug, we can always add backward compatibility later.

**Review by torrocus — DISMISSED** [review]

**Pull request merged**